code/modules/projectiles/projectile/special/goat.dm

```diff
@@ -0,0 +1,18 @@
+/obj/projectile/goat
+	name = "goat"
+	icon = 'icons/mob/animal.dmi'
+	icon_state = "goat"
+	damage = 20
+	damage_type = BRUTE
+	flag = "bullet"
+
+/obj/projectile/goat/on_hit(atom/target)
+	if(target == firer)
+		forceMove(target.loc)
```

Comment:
What exactly is this trying to achieve? Projectiles shouldn't be hitting their firer anyhow. Try to call parent otherwise other important shit doesn't get done.